component/src/main/java/org/wso2/extension/siddhi/io/sqs/util/SQSConstants.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.extension.siddhi.io.sqs.util;
+
+/**
+ * Class to keep constants related to the Sources and Sinks.
+ */
+public class SQSConstants {
+
+    // Option parameter names
+    public static final String QUEUE_NAME = "queue";
+    public static final String ACCESS_KEY_NAME = "access.key";
+    public static final String SECRET_KEY_NAME = "secret.key";
+    public static final String REGION_NAME = "region";
+    public static final String POLLING_INTERVAL_NAME = "polling.interval";
+    public static final String WAITING_TIME_NAME = "waiting.time";
+    public static final String MAX_NUMBER_OF_MESSAGES_NAME = "max.number.of.messages";
+    public static final String VISIBILITY_TIMEOUT_NAME = "visibility.timeout";
+    public static final String PURGE_MESSAGES_NAME = "purge.messages";
```

Comment:
can't we make this **'delete.message**' ?
all the other properties which are related to deleting messages have "delete" instead of "purge"